core/model/modx/processors/security/user/getrecentlyeditedresources.class.php

```diff
@@ -58,8 +58,13 @@ public function initialize()
      */
     public function prepareQueryBeforeCount(xPDOQuery $c)
     {
+        $user = $this->getProperty('user');
         $q = $this->modx->newQuery($this->classKey, ['classKey:IN' => $this->classKeys]);
         $q->select('MAX(id), item');
+        if (!empty($user)) {
+            $q->where(['user' => $user]);
+            $c->where(['user' => $user]);
```

Comment:
Is it required here?

Reply:
Seems like accidentally duplicated lines. Need to leave the first one.

Reply:
> `Seems like accidentally duplicated lines. Need to leave the first one`
Not really, both are required. Please note, there are two different variables **_$q_** and **_$c_**.
I just tried to make as less changes as I could. So I just used the variables which were added before.

Reply:
Ok, I see now. 👍